news/1065.feature

```diff
@@ -0,0 +1 @@
+Add PLD Linux package link to project page. #1065, #1064
```

Comment:
Remove the link from the news file, those aren't needed.
```suggestion
Add PLD Linux package link to project page.
```

Reply:
How else would you have a link to both of them in changelog?

Reply:
No need to link both of them, just the issue itself, or PR if there is no issue. I see that you are using #1065 instead of #1064 in the name. This wouldn't work because the link will point to issue with id 1065, which doesn't exists.

I would recommend to link the rename it to 1064.feature

Reply:
> This wouldn't work because the link will point to issue with id 1065, which doesn't exists.

yes, it will work tecnically, GitHub has a unique id namespace for issues and pull requests, and they both redirect to each other:
- https://github.com/fedora-infra/anitya/issues/1065
- https://github.com/fedora-infra/anitya/pull/1064

I would rather rename it to `PR1065.feature` then, as while it #1065 covers #1064, it doesn't solve it, i.e it's a rather open question for documentation or something.

however, I can't do file rename from GitHub web, so need to check out this locally to make changes, so this will take time.

Reply:
Thanks for doing this